runtime-modules/storage/src/weights.rs

```diff
@@ -18,7 +18,7 @@
 //! Autogenerated weights for storage
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-07-28, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2022-08-23, STEPS: `5`, REPEAT: 2, LOW RANGE: `[]`, HIGH RANGE: `[]`
```

Comment:
Please revert `runtime-modules/storage/src/weights.rs` to the previous state, there's no reason to update this file in this PR